pkgs/games/mindustry/default.nix

```diff
@@ -188,6 +189,7 @@ stdenv.mkDerivation {
       install -Dm644 desktop/build/libs/Mindustry.jar $out/share/mindustry.jar
       mkdir -p $out/bin
       makeWrapper ${jdk}/bin/java $out/bin/mindustry \
+        ${lib.optionalString stdenv.isLinux "--set PATH ${lib.makeBinPath [zenity]}"} \
```

Comment:
Why `--set` and not `--prefix PATH : ${lib.makeBinPath [zenity]}` ?

Reply:
took from [this comment](https://github.com/NixOS/nixpkgs/pull/332917#issuecomment-2272942359). can we ask @fgaz too?

Reply:
Ah. I think `--prefix`-ing the PATH is generally preferrable, as it is more composable with the environment.

Reply:
> Ah. I think `--prefix`-ing the PATH is generally preferrable, as it is more composable with the environment.

seems legit and i won't disagree, asap i'll commit and force-push, in case one of maintainers agree with it. :D

Reply:
I suggested `--set` because `zenity` is the only program mindustry is supposed to execute. It shouldn't need anything else from the environment. `--prefix` should be equivalent to `--set` in this case, so if we want the user to be able to provide a different `zenity` at runtime we should rather use `--suffix`.